plugin/gatsby-node.js

```diff
@@ -4,5 +4,13 @@ exports.pluginOptionsSchema = ({ Joi }) => {
     colors: Joi.array()
       .items(Joi.string().default("#ffffff"))
       .description("Array of hex color values"),
+    duration: Joi.array()
```

Comment:
**issue:** Duration is not an array of numbers, it's a number.

Reply:
duration: Joi.number()